tests/pkg_umorse/main.c

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2017 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ * @file
+ * @brief       Check if the uMorse package builds.
+ *
+ * @author      Sebastian Meiling <s@mlng.net>
+ * @}
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "board.h"
+#include "periph_conf.h"
+#include "xtimer.h"
+#define UMORSE_MSLEEP(m)    xtimer_usleep(m * US_PER_MS)
```

Comment:
if that define is not used by umorse.h, maybe move after all includes and roup with ```CODE_LEN```?

Reply:
well actually it is in `umorse.h` that defines it to be `usleep` by default - which would require posix module in RIOT, right?

Reply:
true, then it is ok